packages/dashboard/src/components/tasks/task-schedule-utils.ts

```diff
@@ -166,8 +166,20 @@ export const apiScheduleToSchedule = (scheduleTask: ApiSchedule[]): Schedule =>
   };
 };
 
-export const getScheduledTaskTitle = (task: ScheduledTask): string => {
-  const shortDescription = getShortDescription(task.task_request);
+export const getScheduledTaskTitle = (
+  task: ScheduledTask,
+  supportedTasks?: TaskDefinition[],
+): string => {
+  let remappedTaskName: string | undefined = undefined;
+  if (supportedTasks) {
+    for (const s of supportedTasks) {
+      if (s.taskDefinitionId === task.task_request.category) {
+        remappedTaskName = s.taskDisplayName;
+      }
```

Comment:
I thought `task.task_request.category` is the top level rmf category? Shouldn't we check the task definition id in the booking label?

Reply:
thanks for catching that, https://github.com/open-rmf/rmf-web/pull/925/commits/d452affaa133634466a30fb774f3af44a2ca417a